src/libraries/System.Runtime.Serialization.Formatters/src/Resources/Strings.resx

```diff
@@ -249,4 +249,7 @@
   <data name="IO_EOF_ReadBeyondEOF" xml:space="preserve">
     <value>Unable to read beyond the end of the stream.</value>
   </data>
+  <data name="BinaryFormatter_SerializationDisallowed" xml:space="preserve">
+    <value>BinaryFormatter serialization and deserialization is disallowed within this application. See https://aka.ms/binaryformatter for more information.</value>
```

Comment:
This feels like it has subject/verb problems.

* "BinaryFormatter serialization and deserialization are"
* "The use of BinaryFormatter is"
* "BinaryFormatter is"